<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintenancePredictiveClass</class>
 <widget class="QMainWindow" name="MaintenancePredictiveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1529</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaintenancePredictive</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-4</y>
          <width>705</width>
          <height>959</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="title_btn0">
           <property name="text">
            <string>数据接入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_0" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QPushButton" name="fileBtn">
              <property name="text">
               <string>本地文件</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="sqlBtn">
              <property name="text">
               <string>数据库</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="title_btn1">
           <property name="text">
            <string>数据统计分析</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>缺失值统计</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>样本不平衡</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>正态分布检验</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>冗余属性识别</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>ADF检验</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>卡方检验</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="title_btn2">
           <property name="text">
            <string>数据预处理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>缺失值填补</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>小波变换</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>onehot编码</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>滤波</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>频域变换</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_13">
              <property name="text">
               <string>差分</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="title_btn3">
           <property name="text">
            <string>特征工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_22">
              <property name="text">
               <string>正则化构造稀疏变量</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_18">
              <property name="text">
               <string>列增加</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_19">
              <property name="text">
               <string>列删除</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_23">
              <property name="text">
               <string>决策树剪枝</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_17">
              <property name="text">
               <string>主成份分析</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_15">
              <property name="text">
               <string>自编码器</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string>相关性分析</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_16">
              <property name="text">
               <string>卡方分布选择特征</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="title_btn4">
           <property name="text">
            <string>机器学习算法</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="clusteringBtn">
              <property name="text">
               <string>聚类</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_24">
              <property name="text">
               <string>决策树分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_26">
              <property name="text">
               <string>随机森林分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_29">
              <property name="text">
               <string>逻辑分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_27">
              <property name="text">
               <string>最近邻分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_25">
              <property name="text">
               <string>支持向量机分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_28">
              <property name="text">
               <string>朴素贝叶斯分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_30">
              <property name="text">
               <string>梯度提升分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_31">
              <property name="text">
               <string>神经网络分类/回归</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="title_btn5">
           <property name="text">
            <string>结果导出</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_20">
              <property name="text">
               <string>数据导出</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_21">
              <property name="text">
               <string>模型导出</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mPButtonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="4,3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="PitureLabel" name="pngLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1529</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PitureLabel</class>
   <extends>QLabel</extends>
   <header>PictrueWgt.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MaintenancePredictive.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="mPButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
